lib/public/Files/IMimeTypeDetector.php

```diff
@@ -73,4 +73,10 @@ public function mimeTypeIcon($mimeType);
 	 * @since 28.0.0
 	 */
 	public function getAllAliases(): array;
+
+	/**
+	 * @return array<string,string>
```

Comment:
The return type does not match what the implementation is doing.

```suggestion
	 * @return array<string, list{string, string|null}>
```